public/stylesheets/admin/style.css

```diff
@@ -0,0 +1,13 @@
+.add-book-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.add-book{
+  padding: 25px;
+  border: 1px solid black;
+  width: 60%;
```

Comment:
Is this inside of the `add-book-container`? If it is, shouldn't you be using relative flex, instead of setting the width?